frontend/src/tests/components/Commons/AdminDisplayTable.test.js

```diff
@@ -0,0 +1,90 @@
+
+import { fireEvent, render, waitFor } from "@testing-library/react";
+import  adminDisplayFixtures  from "fixtures/adminDisplayFixtures";
+import AdminDisplayTable from "main/components/Commons/AdminDisplayTable"
+import { QueryClient, QueryClientProvider } from "react-query";
+import { MemoryRouter } from "react-router-dom";
+import { currentUserFixtures } from "fixtures/currentUserFixtures";
+
+
+describe("AdminDsplayTable tests", () => {
+  const queryClient = new QueryClient();
+  
+  test("renders without crashing for empty table with user not logged in", () => {
+    const currentUser = null;
+
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <AdminDisplayTable admins={[]} currentUser={currentUser} />
+        </MemoryRouter>
+      </QueryClientProvider>
+
+    );
+  });
+  
+  test("renders without crashing for empty table for ordinary user", () => {
+    const currentUser = currentUserFixtures.userOnly;
+
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <AdminDisplayTable admins={[]} currentUser={currentUser} />
+        </MemoryRouter>
+      </QueryClientProvider>
+
+    );
+  });
+  
+
+  test("renders without crashing for empty table for admin", () => {
+    const currentUser = currentUserFixtures.adminUser;
+
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <AdminDisplayTable admins={[]} currentUser={currentUser} />
+        </MemoryRouter>
+      </QueryClientProvider>
+
+    );
+  });
+  
+  test("Has the expected column headers and content for adminUser", () => {
+
+    const currentUser = currentUserFixtures.adminUser;
+
+    const { getByText, getByTestId } = render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <AdminDisplayTable admins={adminDisplayFixtures.threeAdmins} currentUser={currentUser} />
```

Comment:
Same here... and throughout... every use of `admins` should be reviewed and renamed to ... whatever it is you are representing here.

Reply:
Got it